specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.1/TextAnalytics.json

```diff
@@ -60,6 +60,15 @@
             "type": "boolean",
             "required": false
           },
+          {
+            "name": "stringIndexType",
+            "in": "query",
+            "description": "(Optional) This value indicates the index type for which the offsets have to be returned.",
+            "type": "string",
+            "enum" : [ "textelement_v8", "unicodecodepoint", "utf16codeunit" ],
+            "default" : "textelement_v8",
+            "required": false
+          },
```

Comment:
Please see my v3.2-preview.1 [PR](https://github.com/Azure/azure-rest-api-specs-pr/pull/1728/files). It includes v3.1-preview.1 updates that should be incorporated here as well.

Reply:
I have synced this with that PR and this also follows the same conventions now